Probesys/bileto
# tec: Provide a file favicon.ico

## Related issue(s)



https://github.com/Probesys/bileto/issues/764

## How to test manually



1. Open http://localhost:8000/favicon.ico
2. Verify it shows the logo

## Checklist
